src/app/static/src/app/components/downloadIndicator/DownloadIndicator.vue

```diff
@@ -23,7 +23,7 @@
     }
 
     interface Props {
-        filteredData: unknown[] | null
+        filteredData?: unknown[] | null
```

Comment:
```suggestion
        filteredData: unknown[] | null
```

There's no point making it optional from here when you already set required to false.

Reply:
ahhh I have to, so the defineComponentVue2 uses the fact that it is optional prop and enforces that required is false, dont worry next PR we will completely remove all of this business with the defineComponentVue2 so you can safely ignore this one!